libpv/Properties.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, node):
 class TskFileEntryProperties(list):
     def __init__(self, node):
         self.append(wx.propgrid.StringProperty("Full Path",
-            value=u'{}'.format(node.full_path.encode('utf-8', u'replace'))))
+            value=u'{}'.format(node.full_path)))
```

Comment:
Turns out node.full_path already returns as a str

Reply:
Ah, I think full_path is something we create... probably not the best idea to add an attribute an existing class... but this is what happens when you start something and are trying to learn. Woops.